.circleci/local_publish_helpers.sh

```diff
@@ -280,17 +280,16 @@ function runE2eTest {
     fi
 }
 
-function verifyVersionsMatch {
-  internal_cli_package_json_path="./packages/amplify-cli/package.json"
-  npm_cli_package_json_path="./packages/amplify-cli-npm/package.json"
-
-  internal_cli_version=$(jq '.version' $internal_cli_package_json_path)
-  npm_cli_version=$(jq '.version' $npm_cli_package_json_path)
-
-  if [[ $internal_cli_version != $npm_cli_version ]]; then
-    echo "Versions did not match."
-    echo "amplify-cli version: $internal_cli_version"
-    echo "amplify-cli-npm version: $npm_cli_version"
-    exit 1
-  fi
+function checkPackageVersionsInLocalNpmRegistry {
+    cli_internal_version=$(npm view @aws-amplify/cli-internal version)
+    cli_version=$(npm view @aws-amplify/cli version)
+
+    if [[ $cli_internal_version != $cli_version ]]; then
+        echo "Versions did not match."
+        echo "@aws-amplify/cli-internal version: $cli_internal_version"
+        echo "@aws-amplify/cli version: $cli_version"
+        exit 1
+    else
+        echo "Versions matched ($cli_internal_version)"
+    fi
```

Comment:
I suggest to print both versions to console regardless of this if/else condition. So that it's known what this script compares when reading circle ci output.

```suggestion
    cli_internal_version=$(npm view @aws-amplify/cli-internal version)
    cli_version=$(npm view @aws-amplify/cli version)
    echo "@aws-amplify/cli-internal version: $cli_internal_version"
    echo "@aws-amplify/cli version: $cli_version"

    if [[ $cli_internal_version != $cli_version ]]; then
        echo "Versions did not match."
        exit 1
    else
        echo "Versions matched ($cli_internal_version)"
    fi
```